README.md

```diff
@@ -283,7 +283,7 @@ All of the source code required to build intel-device-plugins-for-kubernetes
 is available under Open Source licenses. The source code files identify external Go
 modules used. Binaries are distributed as container images on
 DockerHub. Those images contain license texts under
-`/usr/local/share/package-licenses` and source code under
+`/licenses` and source code under
 `/usr/local/share/package-sources`.
```

Comment:
with `go-licences` the source code is under the same path so this sentence should just say "Those images contain license texts and source code under `/licenses`"